specificationRepositoryConfiguration.json

```diff
@@ -40,6 +40,9 @@
     "azure-resource-manager-schemas": {
       "integrationRepository": "AzureSDKAutomation/azure-resource-manager-schemas",
       "mainRepository": "Azure/azure-resource-manager-schemas"
+    },
+    "azure-powershell": {
+      "mainRepository": "Azure/azure-powershell"
```

Comment:
Please set integrationRepository to https://github.com/AzureSDKAutomation/azure-powershell . And do you consider sdk automation for azure-rest-api-specs-pr?

Reply:
Integration repository has been added. For the private repo, let's skip it for the time being.